docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -1,25 +1,425 @@
+      bareos bareos-bconsole bareos-client bareos-common bareos-database-common bareos-database-postgresql bareos-database-tools bareos-director bareos-filedaemon bareos-storage bareos-tools libgssapi-krb5-2 libk5crypto3 libkrb5-3
+      libkrb5support0
+      15 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
+      Need to get 2557 kB of archives.
+      After this operation, 114 kB of additional disk space will be used.
+      Do you want to continue? [Y/n] Y
+
+.. index::
+   single: update minor bugfix; FreeBSD
+
+.. code-block:: shell-session
+   :caption: Shell example command to update Bareos on FreeBSD
+
+   root@host:~# pkg update --repository Bareos
+   root@host:~# pkg upgrade --repository Bareos
+      Updating Bareos repository catalogue...
+      Bareos repository is up to date.
+      All repositories are up to date.
+      Checking for upgrades (8 candidates): 100%
+      Processing candidates (8 candidates): 100%
+      The following 8 package(s) will be affected (of 0 checked):
+      Installed packages to be UPGRADED:
+            bareos.com-bconsole: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-common: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-database-common: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-database-postgresql: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-database-tools: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-director: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-filedaemon: 21.0.0 -> 21.1.5 [Bareos]
+            bareos.com-storage: 21.0.0 -> 21.1.5 [Bareos]
+      Number of packages to be upgraded: 8
+      1 MiB to be downloaded.
+      Proceed with this action? [y/N]: y
+
+
+.. _section-UpdatePostChecks:
+
+Post update checks
+~~~~~~~~~~~~~~~~~~
+
+.. index::
+   single: Update post checks
+
+After the update, it is recommended to check if any new warnings are raised when starting the daemon, mostly deprecated configuration directives.
+Bareos will mark configuration directives at least for one major release as deprecated, before removing them.
+
+To do so you can use the `-t` flag:
+
+.. code-block:: shell-session
+   :caption: Shell example to check the Bareos configuration
+
+   root@host:~# su - bareos -s /bin/sh -c "bareos-dir -t"
+   root@host:~# su - bareos -s /bin/sh -c "bareos-sd -t"
+   root@host:~# bareos-fd -t
+   There are configuration warnings:
+      * using deprecated keyword PidDirectory on line 19 of file /etc/bareos/bareos-fd.d/client/myself.conf
+
+The same warnings are also shown on a regular start of the daemons.
+
+
+Depending of the operating system and its configuration, you will have to restart the daemons.
+Use your operating system command to do so.
+
+
+.. code-block:: shell-session
+   :caption: Shell command to restart all bareos daemon with systemd on Linux
+
+   root@host:~# systemctl restart bareos-director bareos-storage bareos-filedaemon
+   root@host:~# systemctl status bareos-director bareos-storage bareos-filedaemon
```

Comment:
```suggestion
   root@host:~# systemctl restart bareos-director.service bareos-storage.service bareos-filedaemon.service
   root@host:~# systemctl status bareos-director.service bareos-storage.service bareos-filedaemon.service
```
Isn't it recommended to specify services with the service extension?

Reply:
Never used the .service you don't need it, except when you have several different units with the same name like a .service and .timer or .target for example.